### PR TITLE
fix: [contracts] improve DVM 2.0 comments

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

After spending more time with the DVM 2.0 contracts it became clear that there was still more commenting improvments to make. This PR refines these comments in the DVM2.0 contracts.

### CODE REVIEW

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 1]
```diff
@@ -559,7 +551,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
             voter,
             msg.sender,
             currentRoundId,
-            _getPriceRequest(identifier, time, ancillaryData).pendingRequestIndex,
+            _getPriceRequest(identifier, time, ancillaryData).priceRequestIndex,
```
Comment (chrismaree) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:554]: this was a bug that I found!!! when we emit `VoteRevealed` we need to include the `priceRequestIndex` NOT the `pendingRequestIndex`

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 2]
```diff
@@ -321,15 +318,10 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
...
-    /**
-     * @notice Enqueues a request (if a request isn't already present) for the given identifier, time pair.
-     * @dev Time must be in the past and the identifier must be supported. The length of the ancillary data is limited
-     * such that this method abides by the EVM transaction gas limit.
-     * @param identifier uniquely identifies the price requested. E.g. BTC/USD (encoded as bytes32) could be requested.
-     * @param time unix timestamp for the price request.
-     * @param ancillaryData arbitrary data appended to a price request to give the voters more info from the caller.
-     * @param isGovernance indicates whether the request is for a governance action.
-     */
+    //Enqueues a request (if a request isn't already present) for the given identifier, time and ancillary data. Time
```
Comment (mrice32) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:321]:
```suggestion
    // Enqueues a request (if a request isn't already present) for the given identifier, time and ancillary data. Time
```

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by md0x — APPROVED** [review]
LGTM

**Pull request merged**